db/schema.sql

```diff
@@ -1,16 +1,18 @@
 CREATE TABLE people
 (
-	person_ID 		INTEGER PRIMARY KEY AUTOINCREMENT,
-	email			VARCHAR NOT NULL UNIQUE
+	person_ID 			INTEGER PRIMARY KEY AUTOINCREMENT,
+	email				VARCHAR NOT NULL UNIQUE,
+	givingTo			VARCHAR,

```

Comment:
givingTo is the email? storing it is unnecessary. To retrieve the email you can always do something like:
`
select from.person_ID, from.email, from.givingTo_ID, to.email as givingToEmail
from people from
inner join people to
on from.givingTo_ID = to.person_ID
`

Reply:
Ugh. I know, but I didn't want to. I'll update the database.